index.html

```diff
@@ -22,46 +22,42 @@
             
             <figure class="persona-photo">
-                <img src="img/olana.jpg" alt="">
+                <img src="img/sarima.jpg" alt="">
             </figure>
 
-            <ul class="hats" id="hats-olana">
-                <li class="hat hat-billing">
-                    <i class="haticon haticon-billing"></i> <span class="hat-label">Billing</span>
-                </li>
+            <ul class="hats" id="hats-mark">
                 <li class="hat hat-strategy">
                     <i class="haticon haticon-strategy"></i> <span class="hat-label">Strategy</span>
                 </li>
-                <li class="hat hat-analysis">
-                    <i class="haticon haticon-analysis"></i> <span class="hat-label">Analysis</span>
-                </li>
-                <li class="hat hat-system-health">
-                    <i class="haticon haticon-system-health"></i> <span class="hat-label">System Health</span>
-                </li>
                 <li class="hat hat-implementation">
                     <i class="haticon haticon-implementation"></i> <span class="hat-label">Implementation</span>
                 </li>
+                <li class="hat hat-analysis">
+                    <i class="haticon haticon-analysis"></i> <span class="hat-label">Analysis</span>
+                </li>
             </ul>
 
             <blockquote class="persona-quote">
-                I want to use a web site to build a campaign and watch how it’s doing.
+                Updating a prototype at the click of a button would drastically help turn around time.
             </blockquote>
 
             <div class="persona-details-more">
-                <p>Olana wants to build and run <strong>multiple campaigns, over multiple channels</strong>. He needs to <strong>track the cost</strong> of the campaign, and to <strong>know when things go wrong</strong>, and why. He wants to <strong>see if a campaign is working as expected</strong>, and <strong>see if it's doing well</strong>. He did ICT4D at University a few years ago, so he's <strong>pretty tech-savvy</strong>.</p>
+                <p>Sarima seeks to <strong>understand the needs </strong> of every stakeholder and <strong>design the system</strong> so that it meets stakeholder needs.</p>
 
-                <p>For in-country deploys he wants to <strong>choose the Vumi components that he needs</strong> and just use those. <em>:moonshot:</em></p>
+                <p>Sarima needs to <strong>conduct tests with users</strong> over USSD/SMS/chat on a mobile device. She needs the ability to <strong>easily update her prototype</strong> in the field and in co-design sessions. Making these updates via her mobile phone would be useful. 
+                
+                <p>Sarima needs to see <strong>how users are moving through a flow</strong> and which screens are experiencing high drop-off rates. She also needs to analyze the effect of changes she has made to the system flow. 
```

Comment:
Exporting and post-processing in Excel (for analysis or to report to client) seems important, would it be worth bringing it back?